pgcli/main.py

```diff
@@ -405,6 +420,19 @@ def get_completions(self, text, cursor_positition):
         return self.completer.get_completions(
             Document(text=text, cursor_position=cursor_positition), None)
 
+    def toggle_expanded_output(self):
+        """Callback for the \\x special command"""
+        self.use_expanded_output = not self.use_expanded_output
+        message = u"Expanded display is "
+        message += u"on." if self.use_expanded_output else u"off."
+        return [(None, None, None, message)]
+
+    def toggle_timing(self):
+        self.timing_enabled = not self.timing_enabled
+        message = "Timing is "
+        message += "on." if self.timing_enabled else "off."
+        return [(None, None, None, message)]
+
```

Comment:
Yeah, but then your standalone library starts making pretty strong assumptions about how it's being used. Say the GUI equivalent doesn't support a trivial distinction between compact and extended display (`\x on` and `\x off`) because it has all sorts of crazy gui bells and whistles. Why would it want PGSpecial to support that when it won't be used?